modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -9,20 +9,29 @@ If you plan to install Mule and run it on premises, review these minimum hardwar
 == Minimum Hardware Requirements
 
 * 2 GHz CPU or 1 virtual CPU in virtualized environments
-* 1 GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
+* 1 GB of RAM
 * 4 GB of storage
 
-== Supported Software and Systems
+Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Verify that you use a supported version of Java and a supported operating system before you install Mule.
+== Required Software
 
-Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
+Verify that you use a supported version of Java and a supported operating system before you install Mule.
```

Comment:
Suggestion: I'd recommend reorging the sentence a bit.

"Before you install Mule, verify that..."